views/explore/Explore event Waterfall again.tw

```diff
@@ -0,0 +1,60 @@
+:: Explore event: Waterfall again 
+
+<<set $statistics.waterfall ??= {
```

Comment:
Also probably better this would be to keep under $player.quests as there are other event related stuff already too.
I want to keep "main" variables as less as possible for performance.

Reply:
ok, I'll change the variable